Tests/SwiftLintFrameworkTests/FileNameRuleTests.swift

```diff
@@ -21,6 +22,8 @@ final class FileNameRuleTests: SwiftLintTestCase {
             rule = try FileNameRule(configuration: ["suffix_pattern": suffixPattern])
         } else if let nestedTypeSeparator {
             rule = try FileNameRule(configuration: ["nested_type_separator": nestedTypeSeparator])
+        } else if let fullyQualified {
+            rule = try FileNameRule(configuration: ["fully_qualified": fullyQualified])
```

Comment:
Open to other names here such as `require_fully_qualified`, `require_fully_qualified_names`, etc.

Reply:
I like the last one most.